astroid/brain/brain_typing.py

```diff
@@ -297,6 +297,44 @@ def infer_typing_alias(
     return iter([class_def])
 
 
+def _looks_like_tuple_alias(node: nodes.Call) -> bool:
+    """Return True if call is for Tuple alias.
+
+    In PY37 and PY38 the call is to '_VariadicGenericAlias' with 'tuple' as
+    first argument. In PY39+ it is replaced by a call to '_TupleType'.
+
+    PY37: Tuple = _VariadicGenericAlias(tuple, (), inst=False, special=True)
+    PY39: Tuple = _TupleType(tuple, -1, inst=False, name='Tuple')
+    """
+    return (
+        isinstance(node, nodes.Call)
+        and isinstance(node.func, nodes.Name)
+        and (
+            not PY39
+            and node.func.name == "_VariadicGenericAlias"
+            and isinstance(node.args[0], nodes.Name)
+            and node.args[0].name == "tuple"
+            or PY39
+            and node.func.name == "_TupleType"
```

Comment:
Should we also test the first argument being equal to "tuple"?

Reply:
👍🏻